config/default/forms/contact/person-create.xml

```diff
@@ -899,6 +902,7 @@ DOB ISO: <output value=" /data/person/ephemeral_dob/dob_iso "/></value></text>
       <bind nodeset="/data/person/role_other" type="string" relevant="selected(  /data/person/role ,'other')" required="true()"/>
       <bind nodeset="/data/person/external_id" type="string"/>
       <bind nodeset="/data/person/notes" type="string"/>
+      <bind nodeset="/data/person/user_for_contact/create" type="string" calculate="selected(  /data/person/role ,'chw') or selected(  /data/person/role ,'chw_supervisor')"/>
```

Comment:
Should we make `phone` required if `create` is true? (Since the user creation will fail without a phone)?  Kinda tricky question since the `create_user_for_contacts` transition is not enabled by default (but there is no way to check that from within the form....)

Reply:
Discussed with the @m5r and @tatilepizs.  Decided that the best thing to do is actually just make `user_for_contact/create` dependent on the `phone`.  So, we will only mark `create = 'true'` if a phone number is provided for the contact.